docs/modules/ROOT/partials/k3s_kubeconfig_keycloak.adoc

```diff
@@ -14,4 +14,8 @@ Retrieve the Keycloak password for the `admin` user of the `kubernetes` realm:
 $ terraform output admin_password
 ```
 
-You will use this user and password to log in to applications.
+You will use this user and password to log in:
+
+* Alertmanager
+* Keycloak
+* Prometheus
```

Comment:
And also Grafana

Reply:
nope grafana admin user is not the same as the "admin user in Keycloak". 
I have added a section for Grafana & argoCD -> include::partial$k3s_argocd_grafana.adoc